mw-webapp/src/index.tsx

```diff
@@ -4,6 +4,7 @@ import {BrowserRouter} from "react-router-dom";
 import App from "src/App";
 import reportWebVitals from "src/reportWebVitals";
 import "src/index.scss";
+import "src/firebase";
```

Comment:
Who do you need this live here and  @Ekaterina1994 why it wasn't here before?

Reply:
I test auth before pushing changes. It didn't work without firebase file import.

Reply:
> Who do you need this live here and @Ekaterina1994 why it wasn't here before?

It no need  in index.tsx for get data from realtime db. Maybe for auth need dk. I use import firebase in firebase.ts - it's initialization file with configs

Reply:
@Ekaterina1994 @SergioAJS  please figure out  together does we need it and why we need it :) Before merge